main.js

```diff
@@ -56,40 +58,51 @@ startCompleteButton.addEventListener("click", callStartTimer);
 
 logActivityButton.addEventListener("click", logActivity);
 
+createNewActivityButton.addEventListener("click", returnHome);
+
+window.addEventListener("load", createActivityCard);
+
 // FUNCTIONS
 
+function returnHome(){
```

Comment:
I would say a general rule of thumb would be to avoid reloading the whole page as much as possible. It certainly works based on how you've written the rest of the code which is great! But page reloads are not ideal for a few reasons. 

In general reloading the whole page is not super user friendly, and its a pretty `expensive` action - meaning it takes time, and requires the site to rebuild. In a small app like this its not a huge deal, we have three or four files to load and its a relatively simple landing page. 

Imagine something like gmail that takes a while to load because of the high volume of data. If you had to refresh your inbox every time you closed an email it would take forever. Even for a small app like this if your internet is slow, the reload could take a while or get stuck along the way. 

I would definitely recommend looking into another way to handle the back to home functionality and i'm happy to discuss further why its not a great practice or what other options there are!